src/main/java/uk/gov/hmcts/reform/professionalapi/service/PaymentAccountService.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.professionalapi.service;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.professionalapi.domain.Organisation;
+import uk.gov.hmcts.reform.professionalapi.domain.ProfessionalUser;
+import uk.gov.hmcts.reform.professionalapi.persistence.*;
+
+@Service
+@Slf4j
+@AllArgsConstructor
+public class PaymentAccountService {
+
+    private OrganisationRepository organisationRepository;
+    private ProfessionalUserRepository professionalUserRepository;
+
+    public Organisation findPaymentAccountsByEmail(String email) {
+
+        Organisation organisation = null;
+
+        ProfessionalUser user = professionalUserRepository.findByEmailAddress(email);
+
+        if (user != null) {
+            log.info("user is not empty in payment account service");

```

Comment:
Do we need to log message for email is empty? I don't know what is log.info is doing here

Reply:
I have included these changes in my latest commit. Please review. Thanks!